### PR TITLE
Implemented validation for simple letters count

### PR DESCRIPTION
**Pull request by jeffersonmello**
I needed a simple validation of the number of letters the component did not have this functionality.

I implemented it if you think it's interesting to keep in the project follow the PR.

### CODE REVIEW

**Comment by ArefMozafari** [comment]
Can't you pass all values 0 and use the minLength property to suit your needs?

**Comment by jeffersonmello** [comment]
> Can't you pass all values 0 and use the minLength property to suit your needs?

I my case no, i need validation for min 3 numbers, min 1 upper case letter and min 3 letters.

File: lib/Utilities/Validator.dart [hunk 1]
```diff
@@ -6,6 +6,12 @@ class Validator {
     return password.length >= minLength ? true : false;
   }
 
+  /// Checks if password has at least normal char letter matches
+  bool hasMinNormalChar(String password, int normalCount) {
```
Comment (ArefMozafari) [inline lib/Utilities/Validator.dart:10]: It would be great if you could include a test for this in `Validator_test.dart`

**Review by ArefMozafari — CHANGES_REQUESTED** [review]

**Comment by jeffersonmello** [comment]
@ArefMozafari tests already implemented

**Comment by ArefMozafari** [comment]
Thanks
I will merge this

**Pull request merged**